### ISSUE TITLE
OrderedDict not supported...

### ISSUE THREAD

**Issue by aphillipo** [lyschoening/meteor-ejson-python#2]
I've looked through the code but can't figure out why this does not work:

```
mydict = OrderedDict([(4, 'text 1'), (1, 'text 2'), (23, 'text 3')])
print ejson.dumps(mydict)
```

You don't get the right order:

```
{"1": "text 2", "4": "text 1", "23": "text 3"}
```

Be nice if this was fixed but I'm not sure how!  The default json implementation gets this correct.

**Comment by lyschoening** [comment]
I'll fix this over the weekend if I get the time. The `loads()` code will most likely also need changing.

If you need a temporary fix before, change these lines to return OrderedDict objects https://github.com/lyschoening/meteor-ejson-python/blob/master/ejson/__init__.py#L57:L58

**Comment by aphillipo** [comment]
Ha!  I have a pull request already!  Including test I'll send it!

**Comment by lyschoening** [comment]
Fixed now in 1.1.0 :)

I had hoped to find a good way to support any ordered dictionary class, not just OrderedDict, but the best solution I could come up with is simply to use OrderedDict by default, which is about 50% slower. I might revisit this when Python 3.5 drops.
